doc/lessons_learned.md

```diff
@@ -22,3 +22,4 @@
 * **At NERSC, it matters a lot for throughput where the software is installed.** (Tony)
 * **We need to stay on top of the DMstack (apps and sims) development** both in terms of determining the versions to install but also to sort out problems as they are encountered. This is best done either through [community.lsst.org](http://community.lsst.org) or via GitHub issues (Heather)
 * **Conda installs of DMstack are great, but may not aways be available** In that case, brush up on your use of [lsstsw](https://developer.lsst.io/build-ci/lsstsw.html). (Heather)
+* **PhoSim Centroid files list sourceID incorrectly for large integers** It seems that larger id numbers go outside the default integer length in PhoSim and don't get stored in PhoSim Centroid files correctly. We solved this in Twinkles by making all our id numbers smaller by 10^8 because they can still be unique in our smaller field, but this is still an [open issue in phosim](https://jira.lsstcorp.org/browse/PHOSIM-27).
```

Comment:
I would make it more explicit, something like "In PhoSim, id's are stored as double precision floats.  This means that any number greater than 2^53 ~ 9*10^15 gets rounded off."  That way, we know what the limitations are.

Other than that, go ahead and merge.